composer.json

```diff
@@ -16,10 +16,10 @@
   },
   "require": {
     "php": ">=7.2",
+    "laminas/laminas-filter": "^2.0.0",
```

Comment:
I would prefer to have `spryker/laminas` here

Reply:
The hook repo is too internal IMO for this to be needed, no?

Reply:
IMO it would be easier to handle all laminas dependencies from the one "source of truth" even for internal things.

Reply:
Wouldnt this require more than just this one filter repo?
Either way works for me.

Reply:
Yes, but I don't think this is an issue for an internal tool.

I would prefer to have a spryker dependency here

Reply:
Updated.